soh/src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -1533,7 +1533,13 @@ void KaleidoScope_DrawPages(GlobalContext* globalCtx, GraphicsContext* gfxCtx) {
 
                 gDPSetCombineLERP(POLY_KAL_DISP++, 1, 0, PRIMITIVE, 0, TEXEL0, 0, PRIMITIVE, 0, 1, 0, PRIMITIVE, 0,
                                   TEXEL0, 0, PRIMITIVE, 0);
-                gDPSetPrimColor(POLY_KAL_DISP++, 0, 0, 100, 255, 100, VREG(61));
+                if (CVar_GetS32("gHudColors", 1) == 0) { //Continue prompt cursor colour
+                    gDPSetPrimColor(POLY_KAL_DISP++, 0, 0, 100, 100, 255, VREG(61));
+                } else if (CVar_GetS32("gHudColors", 1) == 1) {
+                    gDPSetPrimColor(POLY_KAL_DISP++, 0, 0, 100, 255, 100, VREG(61));
+                } else if (CVar_GetS32("gHudColors", 1) == 2) {
+                    gDPSetPrimColor(POLY_KAL_DISP++, 0, 0, CVar_GetRGB("gCCABtnPrim", A_BTN_ori).r, CVar_GetRGB("gCCABtnPrim", A_BTN_ori).g, CVar_GetRGB("gCCABtnPrim", A_BTN_ori).b, VREG(61)); //Continue prompt cursor colour
```

Comment:
Small nitpick, you could call CVar_GetRGB once here and store as a variable rather than 3 times

Shouldnt block this from getting merged

Reply:
Wouldn't storing it in a variable cause the color not to update when you change it?

Reply:
I have done it in e5f6c54, if it is not as expected you can revert the commit anyway.